dra-evolution/pkg/api/quota_types.go

```diff
@@ -0,0 +1,46 @@
+package api
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Quota controls whether a ResourceClaim may get allocated.
+// Quota is namespaced and applies to claims within the same namespace.
+type Quota struct {
+	metav1.TypeMeta
+	// Standard object metadata.
+	metav1.ObjectMeta
+
+	Spec QuotaSpec
+}
+
+type QuotaSpec struct {
+	// AllowManagementAccess controls whether claims with ManagementAccess
+	// may be allocated. The default if unset is to deny such access.
+	// +optional
```

Comment:
Add: If multiple quota objects exist and at least one has a true value, access will be allowed.

Reply:
Should I also turn the pointer into a value?

Otherwise we can have one object which explicitly says "true" and another which says "false". When it's a value, that second object isn't possible because the "false" gets omitted.

Reply:
I made it a value.